qt/scientific_interfaces/Indirect/IndirectFitPlotPresenter.cpp

```diff
@@ -240,14 +251,21 @@ void IndirectFitPlotPresenter::setFitSingleSpectrumEnabled(bool enable) {
 }
 
 void IndirectFitPlotPresenter::updatePlots() {
+  HoldRedrawing holdRedrawing(m_view);
```

Comment:
Really like this implementation to stop multiple redraws but then guarantee a redraw when it falls out of scope.